model.go

```diff
@@ -422,7 +422,7 @@ func (m *model) SetBlocks(blocks map[string]string) {
 type ByName []User
 
 func (a ByName) Len() int           { return len(a) }
-func (a ByName) Less(i, j int) bool { return a[i].Name().Id() < a[j].Name().Id() }
+func (a ByName) Less(i, j int) bool { return a[i].Name() < a[j].Name() }
 func (a ByName) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
```

Comment:
These should be removed in favour of just a function for the [`sort.Slice`](https://pkg.go.dev/sort#Slice). That can be done in the future.